### PR TITLE
Update 07_edrs.md

### PR DESCRIPTION
**Pull request by gerbigf**
## WHAT

Fixed the section about provider data plane data transfer in the edr chapter of the management walkthrough and added some more detailed explanation about how the request is build. 

## WHY

Incorrect example

### CODE REVIEW

File: docs/usage/management-api-walkthrough/07_edrs.md [hunk 1]
```diff
@@ -254,14 +254,15 @@ Once the EDR has been negotiated and stored, the data can be fetched in two ways
 
 ## Provider Data Plane
 
-Once the right EDR has been identified using the EDR Management API the current asset/agreement/transfer-process that
-you want to transfer, we can use the `endpoint`, `authorization` to make the request. If the HTTP [Asset](01_assets.md) 
-has been configured to proxy also HTTP verb, query parameters and path segments, they will be forwarded to the backend from the 
-Provider Data Plane:
+Once the right EDR has been identified using the EDR Management API via the correct asset/agreement/transfer-process that
+you want to transfer, we can use the `endpoint` and `authorization` information from the EDRto make the request. The `endpoint` 
+is used as the Host to which the request is sent and the `authorization` is used as the Token in the authorization header.
```
Comment (rafaelmag110) [inline docs/usage/management-api-walkthrough/07_edrs.md:259]:
```suggestion
As in the example bellow, the `endpoint` is used as the Host to which the request is sent and the `authorization` replaces TOKEN in the `Authorization` header.
```

**Review by rafaelmag110 — COMMENTED** [review]
Just some nits...

File: docs/usage/management-api-walkthrough/07_edrs.md (hunk 1 above)
Comment (gerbigf) [reply docs/usage/management-api-walkthrough/07_edrs.md]: Fixed with a different proposal.

**Comment by rafaelmag110** [comment]
Even though its not mentioned anywhere in this repo (i thinkg), the name of the pr should adhere to the Eclipse EDC project contributing guidelines: https://github.com/eclipse-edc/docs/blob/main/CONTRIBUTING.md
At least to make it consistent.
Something like: `docs(edr): <your description>` should do!

**Pull request merged**